### PR TITLE
47 feature check if the repository gives support to the users

### PR DESCRIPTION
**Pull request by FranciscoHeronildo**
## Description

Requests were made to verify that the repository supports users

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

image

### CODE REVIEW

File: hubcare/hubcare_api/hubcare_api/support_views.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        issue_act_float = 0
+
+    support_metric = (
+        readme_int*HEIGHT_README_SUPPORT
+        + issue_temp_int*HEIGHT_ISSUE_TEMPLATE_SUPPORT
+        + license_int*HEIGHT_LICENSE_SUPPORT
+        + description_int*HEIGHT_DESCRIPTION_SUPPORT
+        + code_cond_int*HEIGHT_CODE_OF_CONDUCT_SUPPORT
+        + release_note_int*HEIGHT_RELEASE_NOTE_SUPPORT
+        + issue_act_float*HEIGHT_RATE_ISSUE_ACTIVE_SUPPORT)/16
```
Comment (filipetoyoshima) [inline hubcare/hubcare_api/hubcare_api/support_views.py]: `/16` is hardcoded, just like a magic number. It should be the sum of all the weights of each metric.

And I believe `WEIGHT` is a more appropriate name than `HEIGHT`.

Comment (Brian2397) [reply hubcare/hubcare_api/hubcare_api/support_views.py]: Check the PR again, we already resolved this problem

**Comment by Brian2397** [comment]
The "make style " works ? if I'm not mistaken, pep8 is in trouble

**Comment by cjjcastro** [comment]
I updated the pycodestyle settings and it is now working

**Comment by Brian2397** [comment]
Ok, nos support_metric is a JSON

**Pull request merged**